src/AddressRegistry.Api.BackOffice.Abstractions/Requests/CorrectRegularizedAddressRequest.cs

```diff
@@ -0,0 +1,13 @@
+namespace AddressRegistry.Api.BackOffice.Abstractions.Requests
+{
+    using System.Collections.Generic;
+    using Be.Vlaanderen.Basisregisters.Sqs.Responses;
+    using MediatR;
+    using Newtonsoft.Json;
+
+    public class CorrectRegularizedAddressRequest : CorrectRegularizedAddressBackOfficeRequest, IRequest<ETagResponse>
```

Comment:
We don't implement the sync handlers anymore, so this request is not necessary anymore.